docs/cep/table/index.md

```diff
@@ -74,12 +74,21 @@ The following parameters are configured in a table definition:
 | GLOBAL or LOCAL      | Whether the table is globally or locally replicated. Default is `GLOBAL`. |
 | attribute name   | The schema of the table is defined by its attributes with uniquely identifiable attribute names (`camelCase` is used for attribute names as a convention.)|    |
 | attribute type   | The type of each attribute defined in the schema.  This can be `STRING`, `INT`, `LONG`, `DOUBLE`, `FLOAT`, `BOOL`, or `OBJECT`.     |
```

Comment:
@diana-macrometa - This is an explanation what the attribute name/type is but not params.